lib/src/prices/get_challenge_response.dart

```diff
@@ -0,0 +1,39 @@
+import '../utils/json_helper.dart';
+
+class GetChallengeResponse {
```

Comment:
Duplicate of `ChallengeItem`.
In order to match the other namings, should be named something like `Challenge` or `PriceChallenge` anyway.